### DISCUSSION TITLE
合集自动连播开关和收起

### DISCUSSION THREAD

**Discussion by DarklyAngle** [the1812/Bilibili-Evolved#4037] [功能建议 / Ideas]
希望能加一个合集连播自动关闭的功能，每次都要点一下关闭自动连播。
合集的展开和收起被集成到了“选集区域优化中的展开选集列表”里的，这个功能原本是给番剧用的，后来叔叔搞出了合集，绝大部分番剧没有那么多集无所谓，但是有些合集有非常非常多视频，会占用很大一片区域，希望可以为合集新开一个展开收起

**Comment by Geeyun-JY3** [comment]
> 希望能加一个合集连播自动关闭的功能，每次都要点一下关闭自动连播。

你是想仅合集不自动连播还是多 P 和合集都不自动连播？`传统连播模式`组件可以做到只连播多 P，多 P 和合集都不自动连播关掉破站的`自动连播`（自动切集）就行，破站自己应该会记忆的。

> 合集的展开和收起被集成到了“选集区域优化中的展开选集列表”里的，这个功能原本是给番剧用的，后来叔叔搞出了合集，绝大部分番剧没有那么多集无所谓，但是有些合集有非常非常多视频，会占用很大一片区域，希望可以为合集新开一个展开收起

额你不开 `展开选集列表` 不就不会占用很大一片区域了吗……
而且`选集区域优化`一直都不是针对番剧区做的功能，而且这功能的前身在 [v1](https://github.com/the1812/Bilibili-Evolved/blob/9f452209fed54a6e7a9bdf8cb3c3f7c87bb71e5e/doc/features.md#%E5%B7%A5%E5%85%B7)（只针对多 P 视频）时就已写明对番剧区无效。

**Reply by DarklyAngle** [reply]
希望仅限合集不自动连播，破站的播完暂停只对手动关闭的合集有记忆，新的合集还是会打开自动连播（自动切集）。

**Reply by DarklyAngle** [reply]
合集下面也希望增加一个和视频推荐一样的展开收起
![image](https://user-images.githubusercontent.com/95970563/222451260-a4284171-3061-46a5-90ef-b050175d0d63.png)
![image](https://user-images.githubusercontent.com/95970563/222451319-771e3076-d548-4f73-84c6-369947632572.png)

**Reply by Geeyun-JY3** [reply]
> 希望仅限合集不自动连播，破站的播完暂停只对手动关闭的合集有记忆，新的合集还是会打开自动连播（自动切集）。

`传统连播模式` 不能满足你的需求吗？

**Reply by Geeyun-JY3** [reply]
> 合集下面也希望增加一个和视频推荐一样的展开收起 ![image](https://user-images.githubusercontent.com/95970563/222451260-a4284171-3061-46a5-90ef-b050175d0d63.png) ![image](https://user-images.githubusercontent.com/95970563/222451319-771e3076-d548-4f73-84c6-369947632572.png)

我之前也有过类似想法：https://github.com/the1812/Bilibili-Evolved/issues/692#issue-649834082
目前可以在打开 `展开选集列表` 时，在选集区域的标题上按住 <kbd>Alt</kbd> 键点击可以临时切换展开/收起选集列表：
https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features/features.md#选集区域优化

**Reply by Geeyun-JY3** [reply]
> > 希望仅限合集不自动连播，破站的播完暂停只对手动关闭的合集有记忆，新的合集还是会打开自动连播（自动切集）。
> 
> `传统连播模式` 不能满足你的需求吗？

之前搞错了，目前的`传统连播模式`确实不能满足需求：https://github.com/the1812/Bilibili-Evolved/discussions/3293#discussioncomment-3883344
